utils.py

```diff
@@ -0,0 +1,6 @@
+def get_all_objects_in_collection(collection):
```

Comment:
[Blender already has this exact function](https://docs.blender.org/api/current/bpy.types.Collection.html#bpy.types.Collection.all_objects)